client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:10.15-alpine
```

Comment:
you're testing on node 12 in the CI, you should probably use the same version here

Reply:
Yes indeed - any suggestion on which version we should use in general?